common/concurrency/limits/src/main/java/io/helidon/common/concurrency/limits/AimdLimit.java

```diff
@@ -50,9 +50,11 @@ public class AimdLimit implements Limit, SemaphoreLimit, RuntimeType.Api<AimdLim
     private final AimdLimitConfig config;
     private final AimdLimitImpl aimdLimitImpl;
 
+    private String originName;
+
     private AimdLimit(AimdLimitConfig config) {
         this.config = config;
-        this.aimdLimitImpl = new AimdLimitImpl(config);
+        this.aimdLimitImpl = new AimdLimitImpl(config, originName);
```

Comment:
Is `originName` initialized at this time?

Reply:
Good catch. It's available only when `AimdLimit.init` is invoked which in turn invokes `AimdLimitImpl.initMetrics`. 

Fixed.